src/helpers/xdr/fraction.ts

```diff
@@ -0,0 +1,42 @@
+// This content is copied from js-stellar-base (src/util/continued_fraction.js)
```

Comment:
I put all the XDR helpers from `js-stellar-base` in this folder to make it easier to extract them later if needed.